.pre-commit-config.yaml

```diff
@@ -1,7 +1,94 @@
+default_language_version:
+    python: python3.6
 repos:
 -   repo: https://github.com/psf/black
     rev: stable
     hooks:
     - id: black
-      language_version: python3.6
 
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Would it be worth to also add `check-json`, `check-toml` and `check-yaml` from https://github.com/pre-commit/pre-commit-hooks just in case?

Reply:
I agree.